Create FAQs in EN (#150)
btcven/locha: documentation/en/faqs.md

# FAQ

Welcome to the FAQ of Locha Mesh! Please note that some questions may redirect you to our Documentation for further explaining. 

#### What is a mesh network?

> 

Reviewer: Remove empty line:
```suggestion
```

#### What is a ‘Harpia’ node?
A Harpia node is a Locha Mesh standalone node which can provide services on the network such as an Internet gateway, Bitcoin transactions broadcast, latest blocks data, Electrum Server, a remote monerod, or any other. This device can have a larger antenna plugged, a power amplifier, or even a satellite dish, extending the transmission range in several kilometers. 

#### How does the Locha Mesh work?
The Locha Mesh works by having multiple nodes collaborating with each other on the network, each node implements a routing protocol which makes all nodes connected to the network available through multiple hops. We use IEEE 802.15.4g radios, and a 6LoWPAN/IPv6 stack.

In order to route packets over the mesh, we implemented AODVv2 protocol which only finds a route to a node when we ask for it, so it's relatively efficient to use. You can find more information on the routing protocol in our documentation, see (AODVv2)[./chapter_08.md].

#### What can be transmitted over the Locha Mesh?

> The Locha Mesh uses long-range radio connections in the ISM band (915 MHz in the US and the EU), in this band you can transmit encrypted data at a theoretical speed of ~200 kilobits per second (kbit/s). The data transmitted over the Locha Mesh from the LoChat mobile app or a computer can be any type of file: text, images, audio recordings, offline signed transactions, blocks data, etcetera.

Reviewer: ```suggestion
The Locha Mesh uses long-range radio connections in the ISM band (915 MHz in America, 868 MHz EU), in this band you can transmit encrypted data at a theoretical speed of ~200 kilobits per second (kbit/s). The data transmitted over the Locha Mesh from the LoChat mobile app or a computer can be any type of file: text, images, audio recordings, offline signed transactions, blocks data, etcetera.
```

This can be very dangerous if you are a political dissident, or even if you are just expressing your views in a hostile place, or denouncing abuses, it could get you in trouble, jailed or worse inmediately as they already have the information they need and the way to find you (your phone data connection). 

We believe these situations can be avoided and that communications must be private and safe. To read more on this, see our [Motivation](./chapter_01.md).

#### Where can I buy a Locha Mesh device?
The Turpial device by Locha Mesh can currently be pre-ordered by making a donation [here](https://locha.io/#pre-order).

#### Can other devices join the Locha Mesh network?
Yes! The Locha Mesh protocol is hardware agnostic. This means that there are no “exclusive closed-source” specs for a device to be a node inside the Locha Mesh, other than what we have specified on the guides for a [DIY version of the Turpial for the Locha Mesh](./diy_turpial.md).

#### How many devices running the Locha Mesh protocol are needed to create a mesh?

> 

Reviewer: ```suggestion
```

Two or three devices can alredy be a mesh, but it would not be a resilient communication method. The more devices there are like the Turpial, the DIY version, and Harpia nodes, the better the Locha Mesh will be because it will provide more routes and hops for data to be transmitted over.

#### Do I need to be ‘on-line’ at all times to receive messages?
Yes. The Locha Mesh is completely peer-to-peer, there are no servers or intermediaries, so the messages sent to you over the Locha Mesh while you are disconnected are not stored anywhere waiting for you to connect again. If a user is online, your message will send back an ACK response (acknowledgement response) when has been received by the other party, but if the user is offline you’ll receive a NACK response (negative acknowledgement response). 

For these situations, other nodes can work as ‘servers’ and store messages for an amount of time and then try to deliver them to you in a later time. The owner of the ‘server’ node may charge you for the service provided.

#### I want to contribute. Where do I start?

> 

Reviewer: ```suggestion
```
